targets/targets.json

```diff
@@ -2320,7 +2320,7 @@
         "inherits": ["EFM32"],
         "extra_labels_add": ["EFM32GG", "1024K", "SL_AES"],
         "core": "Cortex-M3",
-        "macros": ["EFM32GG990F1024", "TRANSACTION_QUEUE_SIZE_SPI=4"],
+        "macros": ["EFM32GG990F1024", "TRANSACTION_QUEUE_SIZE_SPI=4", "MBEDTLS_CONFIG_HW_SUPPORT"],
```

Comment:
Can we put this on `EFM32` or do not all `EFM32` have hardware support now?

Reply:
Sure, it can be moved to the EFM32 generic target.